src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -90,7 +90,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->end()
                 ->scalarNode('trust_x_sendfile_type_header')
                     ->info('Set true to enable support for xsendfile in binary file responses.')
-                    ->defaultFalse()
+                    ->defaultValue('%env(bool:default::SYMFONY_TRUST_X_SENDFILE_TYPE_HEADER)%')
```

Comment:
+1, this will prevent this kind of things: https://github.com/dunglas/frankenphp/pull/896/files#diff-6c8777997eada1f39259a3563ecb67c253fbfd91b6d30b99f4933b24c3fe8aa5R31

Reply:
I withdrawed this idea, see https://github.com/symfony/symfony/pull/58162#issuecomment-2331056807